client/src/includes/initTooltips.ts

```diff
@@ -75,37 +78,38 @@ const rotateToggleIcon = {
   },
 };
 
+
 /**
- Default Tippy Tooltips
+ * Default Tippy Tooltips
  */
 export function initTooltips() {
   tippy('[data-tippy-content]', {
     plugins: [hideTooltipOnEsc],
   });
 }
-
 /**
  Actions Dropdown
```

Comment:
```
 * Actions Dropdown initialisation using the Tippy library.
 *
 * @example
 * <div data-button-with-dropdown>
 *  <button data-button-with-dropdown-toggle>...</button>
 *  <div data-button-with-dropdown-content></div>
 * </div>
 */
```

I know this content was already there but it is not correctly formatted JSDOC, also the HTML in this is invalid, maybe let's clean this up a bit.